outbox/shipment-service-quarkus/src/main/resources/application.properties

```diff
@@ -5,7 +5,7 @@ quarkus.datasource.password: mariadbpw
 quarkus.hibernate-orm.database.generation=drop-and-create
 quarkus.hibernate-orm.log.sql=true
 smallrye.messaging.source.orders.type=io.smallrye.reactive.messaging.kafka.Kafka
-smallrye.messaging.source.orders.topic=OrderEvents
+smallrye.messaging.source.orders.topic=outbox.event.order
```

Comment:
Might be worth showing how to actually keep the more custom topic name?

Reply:
I might have screwed this up, so I'm lowering case the aggregate type, so I couldn't find a way to do the regex replacement while upper casing the letter `E` for `OrderEvents`, what I could do is put the topic name to `order.events` for instance or work on a fix for the outbox to be more permissive, but then I don't know how to lower case it later as well! Wdyt?

Reply:
Would it work to use "Order" as aggregate type, and then "${routedByValue}Events" for the `route.topic.replacement` SM option?

Reply:
The final result with this is `orderEvents` due to the lower casing in the transform, I commited so you can see how it looks like!

Reply:
Thanks! Where is the lower-casing coming from? I think I'm still missing a piece of the puzzle to understand.

Reply:
I introduced it here, my line of thought was that I "assumed" that often aggregate types would be camel case, and that would force it to the topic naming, but I actually didn't think that people could be using upper case in their topic names, since what I usually saw is lowercase (including kafka connect official topics, consumer offsets, and such), so I made it to lowercase.
So now that I see it was a bunch of assumptions I want your input on what can be a proper way to deal with it, I tried to find a solution using the regex but couldn't, so not sure if I should just pass it through or make it configurable.
Since it's a dynamic router I'm not sure if even with another router I could achieve it nicely

Reply:
> since what I usually saw is lowercase (including kafka connect official topics, consumer offsets, and such),

I see, that makes sense. Indeed I don't think one can do any changes to casing using Java regexps. So how about going for "order.events" as topic name then (and adjust the aggregate name used by the producer to "order" if needed?